### PR TITLE
Add Docker Action

### PR DESCRIPTION
**Pull request by BraunRudolf**
add docker action file

Modify docker-compose.prod.yml to use environment variables instead of .env files
Solves #7

### CODE REVIEW

**Comment by BraunRudolf** [comment]
Please add secrets/env variables to repository.
Especially variables for DB
`Error: Database is uninitialized and superuser password is not specified.`

File: backend-app/Dockerfile.prod
Comment (bescka) [inline backend-app/Dockerfile.prod:1]: prod using sqlite?

File: frontend-app/Dockerfile.prod
Comment (bescka) [inline frontend-app/Dockerfile.prod:1]: frontend prod build

**Comment by bescka** [comment]
> Please add secrets/env variables to repository. Especially variables for DB `Error: Database is uninitialized and superuser password is not specified.`

Added.

**Review by bescka — APPROVED** [review]

**Comment by BraunRudolf** [comment]
did you miss ``` Error: Database is uninitialized and superuser password is not specified.
filedb_1    |        You must specify POSTGRES_PASSWORD to a non-empty value for the```

**Pull request merged**